### PR TITLE
Docs/improve readme

### LINKED ISSUE #52: [DOCS] improve README.md file
Use your creativity and make improvements to the readme.md file

### PR DESCRIPTION
**Pull request by rohitdasu**
### Fixes Issue

closes #52

### CODE REVIEW

**Comment by AdityaPainuli** [comment]
Everything looks acceptable to me but I think we should use some emojis to make the readme more enjoyable as I see many big companies doc are using emojis. what do you have to suggest on that?

File: README.md [hunk 1]
```diff
@@ -1,60 +1,62 @@
-# projectmate - find your project mate online ⚡
+# ProjectMate - find your project mate online
 
-Live at 🌏 - https://project-mate.vercel.app
+## Demo
+Click here → https://project-mate.vercel.app
```
Comment (RakeshSangem) [inline README.md:4]:
```suggestion
Check out the website: [https://project-mate.vercel.app](https://project-mate.vercel.app)
```

**Comment by RakeshSangem** [comment]
Hey @rohitdasu , One more thing the demo should be after the project description. Could you check that!


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by rohitdasu** [comment]
@RakeshSangem what do you think about icons?

**Comment by RakeshSangem** [comment]
> @RakeshSangem what do you think about icons?

I believe we can use, but not excessively.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by RakeshSangem — APPROVED** [review]
LGTM✌

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by rohitdasu** [comment]
> > @RakeshSangem what do you think about icons?
> 
> I believe we can use, but not excessively.
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

imo we should keep the docs as professional as possible.

**Comment by RakeshSangem** [comment]
> @rohitdasu   imo we should keep the docs as professional as possible.

In that case we can skip them.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Brittle

**Pull request merged**